src/css/styles.css

```diff
@@ -0,0 +1,516 @@
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  padding-top: 30px;
+  padding-bottom: 50px;
+}
+
+.aboutTitle {
+  font-family: 'COCOGOOSE', sans-serif;
+  color: rgb(239, 81, 60);
+  font-size: 25px;
+  font-weight: 900;
+}
+
+.aboutDescription {
+  line-height: 24px;
+  border: 1px solid rgb(216, 219, 221);
+  background-color: #fff;
+  padding: 10px;
+  margin: 20px 20px;
+  text-align: center;
+}
+
+.aboutContact {
+  line-height: 24px;
+  margin: 20px 50px 1px 50px;
+  text-align: center;
+}
+
+.aboutEmail {
+  text-decoration-line: underline;
+  font-weight: 700;
+}
+
+.aboutSecondSection {
+  background-color: white;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 105vh;
+  padding-top: 30px;
+  padding-bottom: 50px;
+}
+
+.aboutLogo {
+  font-size: 12px;
+  padding: 1px 30px 20px 30px;
+  text-align: center;
+}
+
+.aboutPhoto1 {
+  display: block;
+  margin: auto;
+  width: 90%;
+  padding: 5px 5px;
+  background: none;
+  filter: sepia(100%);
+}
+
+.aboutPhoto2 {
+  display: block;
+  margin: auto;
+  width: 90%;
+  padding: 5px 5px;
+  background: none;
+  filter: sepia(100%);
+}
+
+.aboutPartner {
+  font-size: 24px;
+  text-align: center;
+  padding-top: 30px;
+  color: #fff;
+}
+
+.partners {
+  background-color: #3e3e3e;
+  height: 30vh;
+}
+
+.partnerIcons {
+  display: flex;
+  padding-bottom: 20px;
+  margin-right: 20px;
+  justify-content: center;
+}
+
+.rigths {
+  font-size: 12px;
+  padding-left: 30px;
+}
+
+.license {
+  background-color: #fff;
+  color: black;
+  font-size: 24px;
+  padding: 30px 30px;
+  display: flex;
+  align-content: space-around;
+  justify-content: center;
+  height: 150px;
+  align-items: center;
+}
+
+@media (min-width: 768px) {
+  .topBar {
+    background-color: #3e3e3e;
+    color: #fff;
+    height: 36px;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+  }
+
+  .social_icons {
+    background-color: #3e3e3e;
+    display: flex;
+    flex-direction: row;
+    gap: 20px;
+    align-items: center;
+    margin-right: 40px;
+  }
+
+  .navbarItem1 {
+    display: none;
+  }
+
+  .navbarItem2 {
+    display: flex;
+    flex-direction: row;
+    border: 1px solid rgba(0, 0, 0, 0.8);
+    padding: 20px;
+  }
+
+  .itemsDesktopMenu {
+    display: flex;
+    flex-direction: row;
+    gap: 40px;
+    height: 70px;
+    justify-items: flex-end;
+    padding-top: 25px;
+  }
+
+  .itemsDesktopMenu a {
+    text-decoration: none;
+    color: grey;
+  }
+
+  .programDetail {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+  }
+
+  .programList {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 18px;
+    width: 90%;
+    padding-top: 20px;
+    height: 250px;
+  }
+
+  .hero {
+    background-image: url(../images/first-section.png);
+  }
+
```

Comment:
- [ ] On the desktop version kindly have the elements have similar spacing, borders, shadows, and paddings to the original design. Doing so will make your app look even better.

Your Design | Design provided
--|--
![image](https://user-images.githubusercontent.com/29071984/190541252-e77aad40-82ec-4925-b3d8-eca08d88f0d0.png) | ![image](https://user-images.githubusercontent.com/29071984/190541350-2897c0fc-6fe9-402b-a635-4c5d73542d2f.png)